libs/pages/elewa/social-impact/src/lib/pages/components/beyond-business/beyond-business.component.html

```diff
@@ -0,0 +1,7 @@
+<elewa-group-elewa-group-image-and-text-banner
+[imagePlacement]="'left'"
+[titleText]="'Beyond business'"
+[paragraphTexts]="['Some of that youthful energy is channeled beyond business. Through projects such as our coding school in Kakuma refugee camp, training teachers in pastoral Samburu and our open-source coding initiatives providing entry-level opportunities to hundreds of junior engineers', 'These projects don\'t necessarily have an immediate financial return, but their impact can be tangibly measured in the long-term.']"
```

Comment:
Challenge. Set the values as properties in your component and bind to that instead.